packages/engine/src/worker/runner/javascript/mod.rs

```diff
@@ -853,7 +861,11 @@ impl<'m> RunnerImpl<'m> {
         mv8: &'m MiniV8,
         sim_run_id: SimulationShortId,
         mut msg: RunnerTaskMsg,
-    ) -> Result<(TargetedRunnerTaskMsg, Option<Vec<RunnerError>>)> {
+    ) -> Result<(
```

Comment:
There could be a doc comment on `run_task` about what it returns, since it's not obvious from the return type